CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Issues
+- Please use the templates. If none of them is a perfect match, choose the closest one.
+
```

Comment:
We should probably have a section linking to docs on setting up a development environment.